### PR TITLE
Adding SSSOM profile for literals

### LINKED ISSUE #234: SSSOM: Separate profile for Literal Mappings
as discussed in #197 we are now going to provide a basic spec for a literal mapping. This is the suggestion:

- Literals take the place of the "subjects" in the normal sssom
- we ditch all of the subject related metadata columns
- INCOMPLETE ISSUE DONT READ

### PR DESCRIPTION
**Pull request by matentzn**
Fixes #197 
Fixes #234 

- [x] `docs/` have been added/updated if necessary
- [x] `make test` has been run locally
- [x] tests have been added/updated (if applicable)
- [ ] [CHANGELOG.md](https://github.com/mapping-commons/sssom/blob/0.9.0/CHANGELOG.md) has been updated.

This PR adds a new profile to SSSOM for the representation of literal mappings, leaving the default SSSOM intact.

### CODE REVIEW

**Comment by matentzn** [comment]
@udp @cmungall also gave his general nod for this, so I am now convinced we are on the right track.

**Comment by udp** [comment]
@matentzn I'm off sick at the moment - feel free to finish if you like, or I'll be back next week.

**Comment by matentzn** [comment]
It also hit me a bit - I think I would like it if you were to try and finish it :)

**Comment by cmungall** [comment]
note also this previous proposal:

- #155

Which generalizes mapping of literals to mapping of spans in texts

**Comment by matentzn** [comment]
@udp Can you take a look at the OAK suggestion, and look at how our profile overlaps / differs? I think it would be a good thing to avoid profile proliferation, but whatever works is fine!

File: src/sssom_schema/schema/sssom_schema.yaml [hunk 1]
```diff
@@ -576,6 +630,47 @@ classes:
     - other
     - comment
     class_uri: owl:Axiom
+  literal mapping:
+    description: Represents an individual mapping between a a literal and an entity.
```
Comment (matentzn) [inline src/sssom_schema/schema/sssom_schema.yaml:634]:
```suggestion
    description: Represents an individual mapping between a literal and an entity. Note that this schema has been created on 01.08.2023 and is subject to change.
```

**Review by udp — APPROVED** [review]

**Pull request merged**